### ISSUE TITLE
[plugin unplugin-remove-console-debugger] Sourcemap is likely to be incorrect

### ISSUE THREAD

**Issue by Jesse1989pp** [Talljack/unplugin-remove#139]
**Description**
`[plugin unplugin-remove-console-debugger] Sourcemap is likely to be incorrect: a plugin (unplugin-remove-console-debugger) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help`

Ever since version 1.0.0 the recent implementation `removeConsoleLogs` is returning [rollup warnings](https://github.com/rollup/rollup/blob/0b665c31833525c923c0fc20f43ebfca748c6670/src/utils/logs.ts#L929-L936). 

Using the nuxt stack with [nuxt-security](https://github.com/baroshem/nuxt-security), we see this warning popping up a lot of times during compilation. As the warning mentioned it occurs with the sourcemaps "on" and hits the line `document.getElementById('app')!.innerHTML = __UNPLUGIN__${count}\` Inside the vite playground

Been looking into this issue for a little bit, but maybe this makes instant sense to you @Talljack . Will give it another shot soon.

**Workaround**
Currently i added a resolution on this package to version `0.1.7`

**Example:**
You can run th `build:vite` commaned below to see the mentioned rollup warning.
```
diff --git a/package.json b/package.json
index a26da67..6fb03d7 100644
--- a/package.json
+++ b/package.json
@@ -83,7 +83,9 @@
     "lint": "eslint src",
     "lint:fix": "eslint src --fix",
     "play:vite": "npm -C playground/vite run dev",
+    "build:vite": "npm run build && npm -C playground/vite run build",
     "play:webpack": "npm -C playground/webpack run serve",
+    "build:webpack": "npm run build && npm -C playground/webpack run build",
     "prepublishOnly": "npm run build",
     "release": "bumpp && npm publish",
     "start": "esno src/index.ts",
diff --git a/playground/vite/package.json b/playground/vite/package.json
index c9d0a58..9beaf7b 100644
--- a/playground/vite/package.json
+++ b/playground/vite/package.json
@@ -1,8 +1,8 @@
 {
-  "type": "module",
   "private": true,
   "scripts": {
-    "dev": "nodemon -w '../src/**/*.ts' -e .ts -x vite"
+    "dev": "nodemon -w '../src/**/*.ts' -e .ts -x vite",
+    "build": "vite build"
   },
   "devDependencies": {
     "unplugin-remove": "workspace:*",
```

**Comment by Talljack** [comment]
Thank you, I fixed this problem in the version `1.0.2`, I hope you can enjoy it.

**Comment by Jesse1989pp** [comment]
Appreciate that! I made a PR to bump the change in nuxt-security as-well: https://github.com/Baroshem/nuxt-security/pull/420
